src/main/java/seedu/address/model/person/Rsvp.java

```diff
@@ -15,14 +15,16 @@ public class Rsvp {
     public static final String MESSAGE_CONSTRAINTS =
             "RSVP Status should be 'PENDING', 'ACCEPTED' or 'DECLINED'";
 
+    public enum Status {
+        PENDING, ACCEPTED, DECLINED
+    }
+
     /**
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
     public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
Perhaps we should change the regex to ensure that only alphabets are allowed, since the rsvp field should only contain alphabets.